projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
+cd pktd
+./do
+ADD CODE
+
+##Launch pktd
+Now, you can launch pktd:
+
+ADD CODE
+./bin/pktd
+ADD CODE
+
+This should show some output such as the following:
+
+ADD CODE
+2019-08-16 13:47:56.526 [INF] SYNC: Processed 843 blocks in the last 10.01s (843 transactions, height 843, 2019-08-16 05:56:50 +0000 UTC)
+ADD CODE
+
+This tells you that it’s properly syncing the chain. In another window, you can check the status of your node.
+
+ADD CODE
+./bin/pktctl getinfo
+ADD CODE
+
+Check the Block Explorer to see the most recent block.
+
+##Setup the wallet
+First you’ll need to create a wallet
+
+ADD CODE
+./bin/pktwallet --create
+ADD CODE
+
+Then once it is created, launch it
+
+./bin/pktwallet
+Now in another window, you can interact with the wallet. First you might want to create for yourself an address:
+
+ADD CODE
+./bin/pktctl --wallet getnewaddress
+ADD CODE
+
+Or get your current balance:
+ADD CODE
+./bin/pktctl --wallet getbalance
+ADD CODE
+
+To send PKT to somebody, you need to first unlock your wallet. In this example, we’re keeping the wallet unlocked for only 60 seconds, you can change the number at the end to your liking.
+
+ADD CODE
+./bin/pktctl --wallet walletpassphrase <password you used when creating wallet> 60
+ADD CODE
+
+Then send cjd a 10 pkt tip.
+
+ADD CODE
+./bin/pktctl --wallet sendtoaddress pP6Vh6GiL4HsMfcPby5xHQTyBUApd7mewg 10
+ADD CODE
+
+For an exhaustive list of all the RPC calls you can make, use:
+
+ADD CODE
+./bin/pktctl -l
+ADD CODE
+
+  
+  
+{PAGE}
+#About PKT 
+
+##Letter from CJD
+ARPANET Paved the Way
+It's been just over 50 years since the US military issued the first contract to build ARPANET, the network which would eventually evolve into the Internet.
+The year was 1969: Almost everybody got the daily newspaper, typing was a skill mostly confined to secretaries, and if you wanted to communicate with someone far away you would use the telephone. The telephone system itself was something hard to imagine, in most countries there was only one provider, it was illegal to connect your own telephone to the provider's network, and a 5 minute call from New York to Los Angeles cost the equivalent of $18 in today's money.
+It's hard to overstate the impact that the Internet has had on the world. The ability of everyone to become a publisher has led to entirely new forms of journalism and media. Teleconferencing has given rise to remote work and even entirely remote teams. E-commerce and crowdfunding combined with low volume manufacturing have given customers not only incredible choice, but direct input into what products are made. These phenomenon, while wildly futuristic by 1969 standards, would at least have been comprehensible; but open source software projects, decentralized social media, and blockchains are so far removed from that world that there wouldn't even be a basis to explain them.
+The Internet has probably been the biggest single force for individual liberation since the printing press. 
```

Comment:
"individual liberation" is a difficult  claim to make. I've lived in places without Internet and had many, many, many more liberties.  I've also had greater access to market capitalism in places with fewer big companies such as Amazon and Netflix.

What we do get as a result of the Internet is
- greater consumer choice
- less expensive access to substantially more advanced technology
- greater growth potential for big businesses

Don't confuse market opportunities with personal freedom.